packages/paste-design-tokens/tokens/themes/dark/global/text-color.yml

```diff
@@ -75,6 +75,23 @@ props:
     value: "{!palette-gray-0}"
     comment: Strongest primary text.
 
+  # destructive
+  color-text-destructive:
+    value: "{!palette-red-50}"
+    comment: Destructive text.
```

Comment:
thought: should we copy the full `text_contrast_pairing:` blocks from the light theme too?

Reply:
done!